task/02-numbers-tasks.js

```diff
@@ -53,7 +52,7 @@ function getCicleCircumference(radius) {
  *  -3, 3  => 0
  */
 function getAverage(value1, value2) {
-  throw new Error('Not implemented');
+  return value1 === value2 ? value1 : (value1 + value2) / 2;  
```

Comment:
even if two values are identical I don't think it's necessary to have separate cases
`(value1 + value2) / 2;` is enough